src/app.js

```diff
@@ -1 +1,28 @@
 'use strict';
+
+const fs = require('fs');
+
+function copyFiles(source, destination) {
+  if (!source) {
+    // eslint-disable-next-line no-console
+    console.error('No source provided');
+
+    return;
+  }
+
+  if (!destination) {
+    // eslint-disable-next-line no-console
+    console.error('No destination provided');
+
+    return;
+  }
+
+  fs.cp(source, destination, (err) => {
```

Comment:
The `fs.cp` method is not a standard method in Node.js's `fs` module. You might want to use `fs.copyFile` or `fs.copyFileSync` for copying files. Please check the Node.js documentation for the correct method to use.